packages/components/tabs/README.md

```diff
@@ -0,0 +1,144 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - tabs</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-tabs--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.tabs">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.tabs" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.tabs?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.tabs" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.tabs">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.tabs" height="20"/>
+		</a>
+</p>
+
+## ➤ Properties
+
+| Property | Attribute | Type   | Default | Description                                                |
+| -------- | --------- | ------ | ------- | ---------------------------------------------------------- |
+| tabs     | tabs      | array  |         | List of tab objects => properties: {label, icon, selected} |
```

Comment:
I think we need to mention in this Readme that user can use only "ts-tab" inside the component.